src/lib/MongoQueue.ts

```diff
@@ -0,0 +1,94 @@
+import { Model, Schema } from 'mongoose';
+import { Queue, Media, QueueManager } from 'musicca';
+
+export interface IQueue {
+  _id: string;
+  list: Media[];
+}
+
+export const schema = new Schema<IQueue>({
+  _id: String,
+  list: {
+    type: [Object],
+    default: []
+  }
+});
+
+export default class MongoQueue extends Queue {
```

Comment:
Maybe try to not export as default to make it easier to export the whole file exports a.k.a `*` export. (Check my comment on `lib/index.ts`)